CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 
 * The File Effect has been updated to take into account changes in
   `Prelude.File` and to provide a 'better' API.
+* `[a,b..c]` now can be used to generate reverse ranges.
```

Comment:
Apologies in advance, however, range syntax i,e, `[a,b..c]` is syntax sugar. The CHANGELOG entry might be better written with direct reference to the underlying functions `natEnumFromThen` and `natEnumFromTo`, and how this affects the range syntax. Otherwise without knowing how range syntax works, the change might not be easy to understand.

For example:

> `natEnumFromThen` and `natEnumFromTo` have been updated to correctly calculate reverse ranges. Range syntax `[a,b..c]` can be used again to generate reverse ranges.